api/wasm-interpreter/src/lib.rs

```diff
@@ -203,6 +213,42 @@ impl WrappedInstance {
         self.0.highlight(_path.into(), _offset);
         let _ = slint_interpreter::invoke_from_event_loop(|| {}); // wake event loop
     }
+
+    /// THIS FUNCTION IS NOT PART THE PUBLIC API!
+    /// Request information on what to highlight in the editor based on clicks in the UI
+    #[cfg(feature = "highlight")]
+    #[wasm_bindgen]
+    pub fn request_current_item_information(&self, active: bool) {
```

Comment:
should that be called  toggle_design_mode or something like that?

The current name sounds like a getter that would return the item information.

(not only for this function, but also for the property and everywhere)